website/pages/docs/install/cloud-auto-join.mdx

```diff
@@ -95,7 +95,7 @@ $ consul agent -retry-join "provider=azure tag_key=... tag_value=... tenant_id=.
 ```json
 {
   "retry_join": [
-    "provider=azure tag_key=... tag_value=... tenant_id=... client_id=... subscription_id=... secret_access_key=..."
+    "provider=azure tag_name=... tag_value=... tenant_id=... client_id=... subscription_id=... secret_access_key=..."
```

Comment:
Shouldn't this match with the description above? Namely L#88?

Reply:
yes, you are right, i'll change it

Reply:
changed https://github.com/hashicorp/consul/pull/9226/commits/6daf9b1ba1dea414a00e2d92b9cd4b977d9c7129